ir/metadata/specialized_metadata.go

```diff
@@ -2024,10 +2029,11 @@ type DITemplateValueParameter struct {
 	// (optional) Distinct.
 	Distinct bool
 
-	Tag   enum.DwarfTag // optional; zero value if not present.
-	Name  string        // optional; empty if not present.
-	Type  Field         // optional; nil if not present.
-	Value Field         // required.
+	Tag       enum.DwarfTag // optional; zero value if not present.
+	Name      string        // optional; empty if not present.
+	Type      Field         // optional; nil if not present.
+	Value     Field         // required.
+	Defaulted *bool         // optional; nil if not present. TODO: might only need `bool`
```

Comment:
By the looks of it, `false` is the default value used for `defaulted`. So we can switch to `bool` without issue I think.

https://github.com/llvm/llvm-project/commit/c2b437d53d40b6dc5603c97f527398f477d9c5f1#diff-ad303855624f31f09513ee5d6e5b435714d874b6d6c2afcb140b72a4b61a9166R1790

```suggestion
	Defaulted bool         // optional; zero value if not present.
```